dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ConjureBodySerDe.java

```diff
@@ -177,6 +172,11 @@ public T deserialize(Response response) {
             }
         }
 
+        @Override
+        public Optional<String> accepts() {
```

Comment:
Instead of joining the separate accept headers into a single string should we expose a `List<String>`

Reply:
I tried this initially, but found that it maps to adding multiple `Accept` headers each with a single value, but it's most common to use `Accept` header with multiple values. The optional approach allows us to pre-calculate the value.

Reply:
It seems surprising to encode multiple accept values as a header value since in a multimap set up  I would expect each value in the map to be a single header value.

Reply:
I completely agree that multiple accept header key/value pairs would be clean, but I don't think I've seen it used that way in practice. I think we'll reduce odd edge case failures by following the principle of least astonishment.